app/controllers/mentor/knowledges_controller.rb

```diff
@@ -0,0 +1,45 @@
+class Mentor::KnowledgesController < Mentor::BaseController
+  skip_before_action :authenticate_mentor, if: proc { current_mentor}
+  PAGE = 3
+
+  def index
+    @knowledges = Knowledge.page(params[:page]).per(PAGE)
+  end
+
+  def show
+    @knowledge = Knowledge.find(params[:id])
+  end
+
+  def create
+    @knowledge = Knowledge.new
+    if params[:commit] == 'ajax_params'
+      render text: view_context.parse_markdown(params[:knowledge][:content])
+    else
+      Knowledge.create(knowledge_params)
+      redirect_to :action => "index"
```

Comment:
__[RuboCop]__

Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.


<!-- Comment by [SideCI](https://sideci.com) -->